pkgs/by-name/js/jspecify/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchMavenArtifact,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "jspecify";
+  version = "1.0.0";
+
+  src = fetchMavenArtifact {
+    groupId = "org.jspecify";
+    artifactId = "jspecify";
+    version = finalAttrs.version;
+    hash = "sha256-H61ua+dVd4Hk0zcp1Jrhzcj92m/kd7sMxozjUer9+6s=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm444 ${finalAttrs.src.jar} $out/share/java/${finalAttrs.pname}-${finalAttrs.version}.jar
+
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://jspecify.dev";
+    description = "Standard Annotations for Java Static Analysis";
+    license = lib.licenses.asl20;
+    sourceProvenance = with lib.sourceTypes; [ binaryBytecode ];
+    platforms = lib.platforms.unix;
```

```suggestion
    inherit (jre_minimal.meta) platforms;
```

Maybe this is a better way?